.travis/install.sh

```diff
@@ -12,7 +12,7 @@ sudo rm -rf /etc/nginx-sites-enabled/*
 
 # Copy predefined WebDAV config and make sure it works
 sudo cp ./.travis/nginx/webdav.conf /etc/nginx/sites-enabled/webdav.conf
-sudo nginx -t && sudo service nginx reload
+sudo nginx -t && sudo service nginx restart
```

Comment:
AFAIR, ``reload`` leads to update ``nginx`` configuration, so it's should be sufficient to apply changes; ``restart`` should work too, but it takes much time to complete

Reply:
And yes, this file currently not mentioned in ``.travis.yml``,  so this script is unreachable.

Reply:
The restart command is needed to start nginx if it is turned off. Otherwise, the script didn’t work for me locally.